src/intercomparison.py

```diff
@@ -0,0 +1,67 @@
+import sklearn.metrics
+import geopandas as gpd
+import pandas as pd
+import rasterio as rio
+from pyproj import Proj
+from pyproj import transform
+import numpy as np
+import json
+import os
+
+def run_comparison(validationPath: str, cropmapPath: str, validation_projection = None, cropmap_projection: str = None, asJSON = False, subset: str = None):
```

Comment:
For all variable names we've been using lowercase with underscores as per PEP8, it would be good to follow this convention here too (for example `validationPath` -> `validation_path`)